dandiapi/urls.py

```diff
@@ -68,6 +69,11 @@ def to_url(self, value):
 register_converter(DandisetIDConverter, 'dandiset_id')
 urlpatterns = [
     path('api/', include(router.urls)),
+    re_path(
+        r'api/assets/(?P<asset_id>[0-9a-f\-]{36})/metadata/',
```

Comment:
is there a reason to have `metadata` here or could this route be similar to the assets route?

`/assets/<id>` returns metadata
`/assets/<id>/download/` returns pre-signed url

Reply:
I actually think that endpoint should also be `/assets/<id>/metadata`, while `/assets/<id>/` should ideally be reserved for the asset record itself.

If you feel strongly, I can shift mine to match what is there already, but this is definitely a violation of RESTful design principles.

Reply:
we aren't doing that at the other endpoint. i would keep parallel structure rather than different semantics.

Reply:
I'll change this now to preserve the structure, but in the future we should consider changing both endpoints to my suggested design.

Reply:
See 55a00ff.

Reply:
we are being fairly consistent in our design. i know that it differs from some standard practice. but our returns for dandiset versions and assets is the metadata record. this was intentional and i don't think we should change that. and the posts themselves are also the metadata record.